src/SAWScript/Interpreter.hs

```diff
@@ -1740,7 +1740,7 @@ primitives = Map.fromList
     [ "Write the current goal to the given file in SMT-Lib2 format." ]
 
   , prim "w4_offline_smtlib2"  "String -> ProofScript ()"
-    (pureVal offline_smtlib2)
+    (pureVal w4_offline_smtlib2)
```

Comment:
Good catch, although this is somewhat unrelated to the changes to the tutorial elsewhere in the PR. Can you put this change into its own commit?

Reply:
Will do, thanks!